spec/controllers/profile_controller_spec.rb

```diff
@@ -13,23 +15,54 @@
       end
 
       it "get profile if have user" do
-        @request.env["devise.mapping"] = Devise.mappings[:user]
-        user = FactoryBot.create(:user_manager_user)
-        allow(request.env['warden']).to receive(:authenticate!).and_return(@user)
-        allow(controller).to receive(:current_user).and_return(@user)
-        sign_in user
+        usermgr_stub_auth
         get :show
         expect(response.status).to eq(200)
       end
     end
-    
+
+    context 'changing passwords' do
+      it 'errors when new passwords dont match' do
+        usermgr_stub_auth
+        post :change_password, params: { change_password_form: { 'old_password': 'password', 'password': 'wrong', 'password_confirmation': 'right' } }
+        expect(response).to have_http_status(:bad_request)
+        expect(flash.now[:errors]).to eq('Password confirmation doesn\'t match Password')
```

Comment:
Can we localise this?

Reply:
The error comes out of ActiveModel as one of their errors.